system/jlib/jmetrics.hpp

```diff
@@ -44,6 +45,15 @@ enum MetricType
 };
 
 
+struct MetricMetaData
```

Comment:
naming: Should this be named MetricMetaDataEntry/Item?  

Possibly
`    using MetricMetaData = std::vector<MetricMetaDataEntry>;
`